sci-electronics/librepcb/librepcb-0.1.5.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2009-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils xdg
+
+DESCRIPTION="Free EDA software to develop printed circuit boards"
+HOMEPAGE="https://librepcb.org/"
+SRC_URI="https://download.librepcb.org/releases/${PV}/${P}-source.zip"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+BDEPEND="
+	app-arch/unzip
+	dev-qt/linguist-tools:5"
+
+RDEPEND="
+	dev-libs/quazip:=
+	dev-qt/qtcore:5
+	dev-qt/qtconcurrent:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5[ssl]
+	dev-qt/qtopengl:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtsql:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	sys-libs/zlib"
```

Comment:
```
 * librepcb-0.1.5-source.zip BLAKE2B SHA512 size ;-) ...                                     [ ok ]
>>> Unpacking source...
>>> Unpacking librepcb-0.1.5-source.zip to /var/tmp/portage/sci-electronics/librepcb-0.1.5/work
>>> Source unpacked in /var/tmp/portage/sci-electronics/librepcb-0.1.5/work
>>> Preparing source in /var/tmp/portage/sci-electronics/librepcb-0.1.5/work/librepcb-0.1.5 ...
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/sci-electronics/librepcb-0.1.5/work/librepcb-0.1.5 ...
 * Running qmake ...
Info: creating stash file /var/tmp/portage/sci-electronics/librepcb-0.1.5/work/librepcb-0.1.5/.qmake.stash
Reading /var/tmp/portage/sci-electronics/librepcb-0.1.5/work/librepcb-0.1.5/apps/apps.pro
 Reading /var/tmp/portage/sci-electronics/librepcb-0.1.5/work/librepcb-0.1.5/apps/librepcb/librepcb.pro
Project ERROR: Unknown module(s) in QT: svg                                                  [ !! ]
```
So some qt dep needs svg, I'm not sure which one.

We should keep an eye on #19565 and perhaps merge this after quazip change?

Reply:
ebuild needs to add `dev-qt/qtsvg:5` unless it is a bogus dependency.

Reply:
Well now I'm getting 
```
Project ERROR: quazip development package not found                                          [ !! ]

 * Running qmake has failed! (see above for details)
 * This shouldn't happen - please send a bug report to https://bugs.gentoo.org/

 * ERROR: sci-electronics/librepcb-0.1.5::testworld failed (configure phase):
 *   eqmake5 failed
```
after 257b4edee92e6a2aa211ec78251cc317bb6c3323

Could you check what are the workarounds for many packages in https://github.com/gentoo/gentoo/pull/19565/files

Also I don't see qtsvg added here, it seemed to be necessary on my last test run?

Reply:
yes indeed, this has to be fixed. package uses PKGCONFIG to detect system quazip, so look no further than this for inspiration:

https://github.com/gentoo/gentoo/pull/19565/files#diff-def719ef07c52ab037e92ef945c897ce56ea6b467750839f0b7a3395b80a378c
https://github.com/gentoo/gentoo/pull/19565/files#diff-40f346d5b7768a23c642227cbb7ab6c01f068b9552870ca839ab014ff461619cR53

(this is first patching the upstream build system to use quazip1-qt5, then using sed to undo that change in case stable quazip is installed)